### PR TITLE
Joe security v2 palybook

### PR DESCRIPTION
**Pull request by AdiPeret**
**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-6200

## Description
This pull request fixes a deprecated Joe Sandbox Detonate File Playbook implementation.

## Screenshots
Detonate_File_-_JoeSecurity_V2

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [X] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

### CODE REVIEW

**Review by idovandijk — COMMENTED** [review]
Great job! Here are my comments:

1. What happens if the "System" input is empty? Might be good to explain in the description:
image

2. In the outputs, you have correctly outputting the File object, but please also output the rest of the sub-keys of the file (File.SHA1, File.SHA256, etc). The reason is because it makes it easier to see and use from outside of the playbook.

image

3. You have a Timeout input, but you are not using it anywhere. It should be used in this task:
image

After you fix these please let me know and I will review again :)

File: Packs/CommonPlaybooks/ReleaseNotes/2_3_60.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Detonate File - Generic
+
+- Added Joe Sandbox Detonate File V2 and updated output descriptions for VMRay Playbook.
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/ReleaseNotes/2_3_60.md:6]:
```suggestion
Added Joe Sandbox Detonate File V2 and updated output descriptions for VMRay Playbook.
```

Should we mention what additional features the v2 version provides?

**Comment by ShirleyDenkberg** [comment]
@idovandijk @michalgold Doc review completed.

**Review by idovandijk — APPROVED** [review]
Excellent. Once the build passes you can merge!

File: Packs/CommonPlaybooks/ReleaseNotes/2_3_60.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+- Joe Sandbox Detonate File V2 improvements include Added enhanced file outputs and improved file submission mechanism
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/ReleaseNotes/2_3_60.md:7]:
```suggestion
- Joe Sandbox Detonate File V2 improvements include:
   -  Added enhanced file outputs.
   - Improved file submission mechanism.
```

**Comment by ShirleyDenkberg** [comment]
@AdiPeret Is there anything else you need me to review?

**Comment by AdiPeret** [comment]
no, thanks

**Pull request merged**